.gitignore

```diff
@@ -1,3 +1,3 @@
 .DS_Store
 node_modules/
-config.json
+config.json
```

Comment:
Add a new line at the end of every file that has this "small red enter" icon at the end.